doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -709,6 +709,10 @@ Modem libraries
   * Updated the RTT trace backend to allocate the RTT channel at boot, instead of when the modem is activated.
   * Removed support for deprecated RAI socket options ``SO_RAI_LAST``, ``SO_RAI_NO_DATA``, ``SO_RAI_ONE_RESP``, ``SO_RAI_ONGOING``, and ``SO_RAI_WAIT_MORE``.
 
+* :ref:`sms_readme` library:
+
+  * Added "7" as a magic SMS service center address for type approval SIM cards which do not have it set.
```

Comment:
I'm a bit apprehensive about the use of "magic" here. Would something "fallback" work?

Reply:
Fallback sounds fine. Changed.